File: 01-March-2024/32- Count Pairs Whose Sum is Less than Target/s.php
class Solution {

/**
 * @param Integer[] $nums
 * @param Integer $target
 * @return Integer
 */
function countPairs($nums, $target) {
    $counter = 0 ;
    for($i = 0 ; $i < count($nums) ; $i++)
    {
        for($j = $i + 1 ; $j < count($nums) ; $j++)
        {
            if( $nums[$i] + $nums[$j] <  $target )
            {
                $counter++;
            }
        }
    }
    return $counter;
}
}

File: 01-March-2024/46- Count Equal and Divisible Pairs in an Array/s.rs
impl Solution {
    pub fn count_pairs(nums: Vec<i32>, k: i32) -> i32 {
        let mut counter:i32 = 0;
        for i in 0..nums.len()
        {
            for j in i+1..nums.len()
            {
                if nums[i] == nums[j]  && (i * j) as i32 % k == 0 {
                    counter += 1;
                }
            }
        }
        return counter;
    }
}